packages/perseus/src/widgets/numeric-input/numeric-input.testdata.ts

```diff
@@ -134,6 +175,7 @@ export const multipleAnswersWithDecimals: PerseusRenderer = {
                         value: 12.2,
                         simplify: "required",
                         message: "",
+                        answerForms: ["decimal"],
```

Comment:
Realized that these should be specifying their answerForms, just for consistency.